CODE_OF_CONDUCT.md

```diff
@@ -1,5 +1,149 @@
+  * Trolling, insulting/derogatory comments, and personal or political attacks.
+  * Public or private harassment, deliberate intimidation, or threats.
+  * Publishing others' private information, such as a physical or electronic address, without explicit permission. This includes any sort of "outing" of any aspect of someone's identity without their consent.
+  * Publishing private screenshots or quotes of interactions in the context of this project without all quoted users' *explicit* consent.
+  * Publishing of private communication that doesn't have to do with reporting harrassment.
+  * Any of the above even when [presented as "ironic" or "joking"](https://en.wikipedia.org/wiki/Hipster_racism).
+  * Any attempt to present "reverse-ism" versions of the above as violations. Examples of reverse-isms are "reverse racism", "reverse sexism", "heterophobia", and "cisphobia".
+  * Unsolicited explanations under the assumption that someone doesn't already know it. Ask before you teach! Don't assume what people's knowledge gaps are.
+  * [Feigning or exaggerating surprise](https://www.recurse.com/manual#no-feigned-surprise) when someone admits to not knowing something.
+  * "[Well-actuallies](https://www.recurse.com/manual#no-well-actuallys)"
+  * Other conduct which could reasonably be considered inappropriate in a professional or community setting.
+
+## Scope
+
+This Code of Conduct applies both within spaces involving this project and in other spaces involving community members. This includes the repository, its Pull Requests and Issue tracker, its Twitter community, private email communications in the context of the project, and any events where members of the project are participating, as well as adjacent communities and venues affecting the project's members.
+
+Depending on the violation, the moderators may decide that violations of this code of conduct that have happened outside of the scope of the community may deem an individual unwelcome, and take appropriate action to maintain the comfort and safety of its members.
+
+### Other Community Standards
+
+As a project on GitHub, this project is additionally covered by the [GitHub Community Guidelines](https://help.github.com/articles/github-community-guidelines/).
+
+Enforcement of those guidelines after violations overlapping with the above are the responsibility of the entities, and moderation may happen in any or all of the services/communities.
+
+## Moderator Moderation Process
+
+Once the moderators get involved, they will follow a documented series of steps and do their best to preserve the well-being of project members. This section covers actual concrete steps.
+
+### Contacting Moderators
+
+Official moderation is done by the Ayo.js Moderation Team.
+
+* Ping [`@ayojs/moderation`](https://github.com/orgs/ayojs/teams/moderation/members) on the GitHub issue.
+* Ping `@moderation` in the [Ayo.js Discord](https://discord.gg/hCgptwH).
+* TODO: email?
```

Comment:
Can we have an explicit option of contacting one trusted member of the moderation team privately?

Reply:
i think this should be possible once #70 has landed?

Reply:
I'd like this, yes.